control-plane/agents/src/bin/ha/node/detector.rs

```diff
@@ -147,6 +155,22 @@ impl PathRecord {
         }
         self.state = PathState::Good;
     }
+
+    fn report_deleting(&mut self) {
+        let to = PathState::Removing;
+        if self.state == to {
+            return;
+        }
+        tracing::info!(
+            from=%self.state,
+            %to,
+            target=self.nqn,
+            path=self.path,
+            "Target path is deleting",
+        );
+        //self.event(EventAction::NvmePathDeleting).generate();
```

Comment:
Is this needed for eventing?

Reply:
Need a change on dependencies to add path deleting event